x-pack/plugins/infra/server/graphql/nodes/resolvers.ts

```diff
@@ -35,7 +36,16 @@ export const createNodeResolvers = (
     async map(source, args, { req }, info) {
       const { metrics, path } = extractPathsAndMetrics(info);
       const { groupBy, nodeType } = extractGroupByAndNodeFromPath(path);
-
+      switch (nodeType) {
```

Comment:
What if we crated a function called `countNode(nodeType: InfraNodeType)` on UsageCollector. Then we would just have this switch statement in one place instead of two.

Reply:
Good point.